nvim/queries/tlaplus/highlights.scm

```diff
@@ -112,44 +193,13 @@
 ] @punctuation.bracket
 [
   ","
-  "."
   ":"
+  "."
   "!"
+  ";"
+  (bullet_conj)
+  (bullet_disj)
 ] @punctuation.delimiter
-(bullet_conj) @punctuation.special
-(bullet_disj) @punctuation.special
-(prev_func_val) @punctuation.special
-(placeholder) @punctuation.special
```

Comment:
Should these two tokens have any defined highlighting?

Reply:
fixed